aten/src/ATen/native/quantized/library.cpp

```diff
@@ -152,6 +152,14 @@ TORCH_LIBRARY(quantized, m) {
   m.def(TORCH_SELECTIVE_SCHEMA("quantized::linear_relu_dynamic(Tensor X, __torch__.torch.classes.quantized.LinearPackedParamsBase W_prepack, bool reduce_range=False) -> Tensor Y"));
   m.def(TORCH_SELECTIVE_SCHEMA("quantized::linear_dynamic_fp16(Tensor X, __torch__.torch.classes.quantized.LinearPackedParamsBase W_prepack) -> Tensor Y"));
   m.def(TORCH_SELECTIVE_SCHEMA("quantized::linear_relu_dynamic_fp16(Tensor X, __torch__.torch.classes.quantized.LinearPackedParamsBase W_prepack) -> Tensor Y"));
+  // Corresponding pattern:
```

Comment:
can we clearly explain what the asterisk means here?  I understand it based on your PR comments, but it's not clear from the diagram.

can we also add an "after" state?

Reply:
since this function is also not consistent with other functions in this file, it would be great to add expected input and output dtypes to the docblock.  We should specify that the input is fp32, and we have a packed weight with int8 format packed for xnnpack/qnnpack/fbgemm, and the output is fp32.